settings_screen.tcl

```diff
@@ -0,0 +1,57 @@
+
```

Comment:
@silasg this file :)

Reply:
Wow thanks! I never was aware I committed this file. Probably it was there before different UIs has been introduced and it was still left on my local drive and accidental committed weeks ago. Sorry for the mess!